Patch created by brosx "diffPackage" tool

usage:
    diffPackage <tarName> <pkgName> <outName> <patchName> <files>

actual:
    diffPackage "acl-2.2.52.src.tar.gz" "acl" "host-acl" "disable-long-file-support" "\
        chacl/chacl.c \
    ";


===============================================================================
--- a/chacl/chacl.c	2013-04-30 17:20:43.000000000 +0200
+++ b/chacl/chacl.c	2018-01-03 18:06:57.000000000 +0100
@@ -32,6 +32,10 @@
 #include <locale.h>
 #include "config.h"
 
+	// no dirent64 support on OSX-10
+	#define dirent64 dirent
+	#define readdir64 readdir
+
 static int acl_delete_file (const char * path, acl_type_t type);
 static int list_acl(char *file);
 static int set_acl(acl_t acl, acl_t dacl, const char *fname);
